clients/quarkus/prometheus-client/prometheus-client-config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "modelPackage": "org.candlepin.subscriptions.prometheus.model",
+  "apiPackage": "org.candlepin.subscriptions.prometheus.resources",
+  "invokerPackage": "org.candlepin.subscriptions.prometheus",
+  "groupId": "org.candlepin",
```

Comment:
Please use `groupId` and package prefix of `com.redhat.swatch`; it's not actually used, but is confusing to see, see `buildSrc/src/main/groovy/swatch.quarkus-rest-client-conventions.gradle` for where its overridden.

Reply:
This configuration was actually not used. Removed.